app/models/user.rb

```diff
@@ -5,4 +5,6 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable
   has_one :fridge, dependent: :destroy
   validates :phone_number, format: { with: /\A(?:\+?\d{1,3}\s*-?)?\(?(?:\d{3})?\)?[- ]?\d{3}[- ]?\d{4}\z/ }
+  has_many :favorite_recipes
+  has_many :recipes, through: :favorite_recipes
```

Comment:
i am not sure about that - should it not be: has_many :recipes (as in the ones that he created) additionally to the current line nine?

also: still the old (better :D ) spelling